w08h03/test/pgdp/datastructures/UnitTest.java

```diff
@@ -83,7 +85,7 @@ public void numericTest() {
 
 	private String generateString(Random rnd) {
 		StringBuilder s = new StringBuilder();
-		rnd.ints(rnd.nextInt(100), 97, 122).mapToObj(i -> (char) i).forEach(c -> s.append(c));
+		rnd.ints(rnd.nextInt(100), 97, 122).mapToObj(i -> (char) i).forEach(s::append);
```

Comment:
oops, this was a recommendation from my editor and I clicked it. I guess it's fine to keep in tho...

Reply:
👍